docs/markdown/Using Pants/concepts/goals.md

```diff
@@ -20,71 +20,58 @@ You'll see more goals activated as you activate more [backends](doc:enabling-bac
 ```
 
 You can also run multiple goals in a single run of Pants, in which case they will run sequentially:
 
 ```bash
-# Format all code, and then lint it:
-❯ ./pants fmt lint ::
+# Format all code, and then test it:
+❯ ./pants fmt test ::
 ```
 
-Finally, Pants supports running goals in a `--loop`: in this mode, all goals specified will run sequentially, and then Pants will wait until a relevant file has changed to try running them again.
+Finally, Pants supports running goals in a `--loop`. In this mode, all goals specified will run
+sequentially, and then Pants will wait until a relevant file has changed to try running them again.
 
 ```bash
-# Re-run typechecking and testing continuously as files or their dependencies change:
-❯ ./pants --loop check test project/app_test.py
+# Re-run linters and testing continuously as files or their dependencies change:
+❯ ./pants --loop lint test project/app_test.py
 ```
 
 Use `Ctrl+C` to exit the `--loop`.
 
 # Goal arguments
 
-Some simple goals—such as the `roots` goal—do not require arguments. But most goals require some arguments to work on. 
-
-For example, to run the `count-loc` goal, which counts lines of code in your repository, you need to provide a set of files and/or targets to run on:
-[block:code]
-{
-  "codes": [
-    {
-      "code": "$ ./pants count-loc '**'\n───────────────────────────────────────────────────────────────────────────────\nLanguage                 Files     Lines   Blanks  Comments     Code Complexity\n───────────────────────────────────────────────────────────────────────────────\nPython                      13       155       50        22       83          5\nBASH                         2       261       29        22      210         10\nJSON                         2        25        0         0       25          0\nPlain Text                   2        43        1         0       42          0\nTOML                         2        65       14        18       33          0\n...",
-      "language": "text",
-      "name": "Shell"
-    }
-  ]
-}
-[/block]
-
-[block:callout]
-{
-  "type": "info",
-  "title": "Quoting file patterns",
-  "body": "Note the single-quotes around the file pattern `'**'`. This is so that your shell doesn't attempt to expand the pattern, but instead passes it unaltered to Pants."
-}
-[/block]
-## File arguments vs. target arguments
+Most goals require arguments to know what to work on. 
 
-Goal arguments can be of one of two types:
+You can use several argument types:
 
-- *File arguments*: file paths and/or globs.
-- *Target arguments*: addresses and/or address globs of [targets](doc:targets).
+| Argument type                   | Semantics                                   | Example                         |
+|---------------------------------|---------------------------------------------|---------------------------------|
+| File path                       | Match the file                              | `./pants test project/tests.py` |
```

Comment:
I could add `and match all targets that "own" the file`, but that's probably confusing. Essentially, we match the file.